db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/StateMachines/ProvisionRequestApproval.class/__methods__/validate_request.rb

```diff
@@ -51,10 +81,7 @@
 
 # Validate max_cpus if not nil or empty
 unless max_cpus.blank?
-  desired_cpus = prov_resource.get_option(:number_of_cpus).to_i
-  if desired_cpus.zero?
-    desired_cpus = prov_resource.get_option(:number_of_sockets).to_i * prov_resource.get_option(:cores_per_socket).to_i
-  end
+  desired_cpus = requested_cpu(prov_resource, desired_nvms)
```

Comment:
@tinaafitz The new `requested_cpu` method does not retain the logic to look at `:number_of_sockets` and `:cores_per_socket` when `:number_of_cpus` is empty.  

The `:number_of_cpus` field is only used in the provisioning dialog for SCVMM, all other Infra dialogs use `:number_of_sockets` and `:cores_per_socket.

Since this method is in the Cloud namespace will that ever come into play or are you trying to keep them in sync?